docs/application/native/api/iot-headed/5.0/group__CAPI__OAUTH2__MODULE.html

```diff
@@ -114,7 +114,7 @@ <h2><a class="anchor" id="CAPI_OAUTH2_MODULE_FEATURE"></a>
 </p>
 <p>To ensure your application is only running on the device with specific features, please define the features in your manifest file using the manifest editor in the SDK.<br/>
 </p>
-<p>More details on featuring your application can be found from <a href="https://developer.tizen.org/development/tizen-studio/native-tools/configuring-your-app/manifest-text-editor#feature"><b>Feature List</b>.</a>a&gt;</p>
+<p>More details on featuring your application can be found from <a href="https://docs.tizen.org/application/tizen-studio/native-tools/manifest-text-editor#feature-element"><b>Feature List</b>.</a>a&gt;</p>
```

Comment:
Could you delete `a&gt;` after `<b>Feature List</b>.</a>`?
The 6.0 version also has the same content, please modify it.

Reply:
oauth2_doc.h should be fixed to remove unnecessary a&gt;
I've added it to issue page with No.59.

In this case, I've fixed html directly and pushed changes.